inlong-sort/sort-connectors/cdc-base/src/main/java/org/apache/inlong/sort/cdc/base/source/assigner/HybridSplitAssigner.java

```diff
@@ -115,15 +115,23 @@ public Optional<SourceSplitBase> getNext() {
             // stream split assigning
             if (isStreamSplitAssigned) {
                 // no more splits for the assigner
+                LOG.trace(
+                        "No more splits for the SnapshotSplitAssigner. StreamSplit is already assigned.");
                 return Optional.empty();
             } else if (snapshotSplitAssigner.isFinished()) {
                 // we need to wait snapshot-assigner to be finished before
                 // assigning the stream split. Otherwise, records emitted from stream split
                 // might be out-of-order in terms of same primary key with snapshot splits.
                 isStreamSplitAssigned = true;
-                return Optional.of(createStreamSplit());
+                StreamSplit streamSplit = createStreamSplit();
+                LOG.trace(
+                        "SnapshotSplitAssigner is finished: creating a new stream split {}",
+                        streamSplit);
```

Comment:
If some logs are not important, they can be removed. Excessive logging can impact performance.